src/fenicsxconcrete/finite_element_problem/concrete_thermo_mechanical.py

```diff
@@ -85,7 +87,8 @@ def default_parameters() -> tuple[Experiment, dict[str, pint.Quantity]]:
         default_parameters = {
             "igc": 8.3145 * ureg("J/K/mol"),
             "rho": 2350.0 * ureg("kg/m^3"),
-            "thermal_cond": 2.0 * ureg("W/(m*K)"),
+            "g": 9.81 * ureg("m/s^2"),
+            "thermal_cond": 2.0 * ureg("W/(m^3*K)"),  # TODO Check with Sjard
```

Comment:
According to all documents that @eriktamsen has written and to wikipedia, this should be $W/(m\cdot K)$
```suggestion
            "thermal_cond": 2.0 * ureg("W/(m*K)"),  # TODO Check with Sjard
```